v-next/hardhat/src/internal/builtin-plugins/solidity-test/index.ts

```diff
@@ -16,8 +21,12 @@ const hardhatPlugin: HardhatPlugin = {
         defaultValue: 60 * 60 * 1000,
       })
       .addFlag({
-        name: "noCompile",
-        description: "Don't compile the project before running the tests",
+        name: "force",
+        description: "Force compilation even if no files have changed",
+      })
+      .addFlag({
+        name: "quiet",
+        description: "Makes the compilation process less verbose",
       })
```

Comment:
Agreed with removing `noCompile`.

Now, I'm not sure if these new flags are valuable tbh. And with this names they are definitely confusing, as they are flags to the compilation process, but belong to a test task. I think we should do the same and remove them, at least for now.

Reply:
OK, no worries. I'll remove them for now.

Reply:
I think removing `noCompile` broke the main `test` task, which always sets that param to true for all test subtasks.

Reply:
Ay! You're right! Sorry about that. I'm on it!